src/components/MenuFlyout/index.tsx

```diff
@@ -95,25 +95,22 @@ const MenuFlyout = (props: {
 }) => {
   useOnClickOutside(props.node, props.isOpen && !isMobile ? props.toggle : undefined)
   if (!props.isOpen) return null
-  return (
-    <>
-      <BrowserView>
-        <BrowserStyle hasArrow={!!props.hasArrow} customStyle={props.browserCustomStyle}>
+  if (isMobile)
```

Comment:
it fixed this issues https://www.notion.so/kybernetwork/Display-double-Preferences-pop-up-on-iPad-bf5de482cfe5495b886880ee12549129